_data/vulnerability_scanning_tools.yml

```diff
@@ -38,8 +38,8 @@
   owner: Arachni	
   license: Free for most use cases	
   platforms: Most platforms supported
-- title: AVDS
-  url: https://www.scanmyserver.com/
+- title: beSECURE (Formally: AVDS)

```

Comment:
Formally or formerly? (ex: official or before/previously)

Reply:
Sorry, I mean Formerly (previously - the name changed)

Before:
https://beyondsecurity.com/avds.html

Now:
https://beyondsecurity.com/solutions/besecure.html